tests/integration/testdata/buildcmd/Dotnet8/HelloWorld.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Amazon.Lambda.Core" Version="1.0.0" />
```

Comment:
Probably doesn't matter but this test Lambda project is using old references including the older Serialization package. From a test point of view it probably doesn't matter but it might be worth updating so people don't copy and paste this somewhere else.

Reply:
Updated. Also removed the extra usings from the function